src/pages/Work/Tabs/TechnicalTab.tsx

```diff
@@ -13,15 +13,18 @@ export class TechnicalTab extends WorkById {
   renderElement(resource: Api.WorkById.Response, headers: Headers) {
     if (!resource) return <div className="technical-tab">Could not load technical information.</div>
 
-    if (!resource.timestamp)
+    if (!resource.timestamp && !resource.anchor)
       return (
         <div className="technical-tab">Technical information not available. This work may be pending confirmation.</div>
       )
 
     return (
       <div className="technical-tab">
         <table>
-          <tbody>{Object.entries(resource.timestamp).map(([key, value]) => this.renderEntry(key, value))}</tbody>
+          <tbody>{Object.entries(resource.timestamp ?
```

Comment:
Same here. `timestamp`'s no longer used. Aren't we free to make backwards-incompatible changes in `master` since it no longer gets deployed to explorer.po.et?

Reply:
Yeah we should, but we need unit tests for it, for regression testing.